.gitignore

```diff
@@ -59,6 +59,8 @@ asn1_*.[cx]
 /tags
 /test-driver
 /ylwrap
+/build
```

Comment:
I'd rather not ignore this.  I like being able to do `git clean -fdx` to get a clean workspace.